docs/source/user_guide/automl.ipynb

```diff
@@ -358,27 +358,30 @@
    "source": [
     "from evalml import AutoMLSearch\n",
     "from evalml.demos import load_fraud\n",
-    "from evalml.pipelines.components.utils import get_estimators\n",
+    "from skopt.space import Categorical\n",
     "from evalml.model_family import ModelFamily\n",
-    "from evalml.pipelines.utils import make_pipeline\n",
     "import woodwork as ww\n",
     "\n",
     "X, y = load_fraud(n_rows=1000)\n",
     "\n",
     "# example of setting parameter to just one value\n",
-    "custom_hyperparameters = {'Imputer': {\n",
-    "    'numeric_impute_strategy': ['mean']\n",
+    "pipeline_parameters = {'Imputer': {\n",
```

Comment:
nit-pick: Shouldn't these be called hyperparameters?

Reply:
It is, but it defines both the hyperparameter and parameter ranges. Since I discussed it above this cell with 
```
To limit the search over certain hyperparameter ranges, you can specify a pipeline_parameters argument with your pipeline parameters.
These parameters will also limit the hyperparameter search space. 
```
I figured it would be fine to call it pipeline_parameters, since this was what I would be passing in. Would it make more sense to name it `pipeline_hyperparamters` instead?

Reply:
Changed!